pkgs/top-level/perl-packages.nix

```diff
@@ -18720,6 +18720,8 @@ let
       url = mirror://cpan/authors/id/G/GB/GBARR/TimeDate-2.30.tar.gz;
       sha256 = "11lf54akr9nbivqkjrhvkmfdgkbhw85sq0q4mak56n6bf542bgbm";
     };
+    # https://rt.cpan.org/Public/Bug/Display.html?id=124509
+    patches = [ ../development/perl-modules/timedate-2020.patch ];
```

Comment:
Can we please use ```fetchPatch``` from https://github.com/atoomic/perl-TimeDate/commit/4b67ccbdc1846620470ca524a5f3e2afd7b33f66 ?

Reply:
I agree using `fetchPatch` would be better. However let's do this in a separate PR against `master` and cherry-pick it here. Ideally most commits in release branches are just cherry picks.

Reply:
See: https://github.com/NixOS/nixpkgs/pull/78765